reference-architecture/aws-ansible/README.md

```diff
@@ -58,6 +58,9 @@ The default region is us-east-1 but can be changed when running the ose-on-aws s
 ### AMI ID
 The AMI ID may need to change if the AWS IAM account does not have access to the Red Hat Cloud Access gold image or if deploying outside of the us-east-1 region.
 
+### Containerized Installation
+The option to install OpenShift in containers is possible by providing the trigger --containerized=true.
```

Comment:
Wording is weird. Maybe something like:
The trigger(selection) containerized=true , provides the option to install OpenShift using Contianers.